list_missing_dates.py

```diff
@@ -0,0 +1,47 @@
+import sys
+import requests, json
+
+def get_refdes_list():
+    http = 'http://uframe-3-test:12576/sensor/inv/'
+    r = requests.get(http)
+    refdes_json = r.json()
+    site_list = []
+    refdes_list = []
+    
+    for item in refdes_json:
+        site = item
+        site_http = http + item + '/'
+        site_node = requests.get(site_http)
+        
+        for item1 in site_node.json():
+            node = item1
+            node_http = site_http + item1 + '/'
+            site_node_inst = requests.get(node_http)
+            
+            for item2 in site_node_inst.json():
+                inst = item2
+                refdes_list.append(site + '-' + node + '-' + inst)
+                print site + '-' + node + '-' + inst
+    return refdes_list
+
+
+def get_missing_data_list(refdes):
+    http = "http://uframe-3-test.intra.oceanobservatories.org:9000/available/" + refdes
```

Comment:
Think about the purpose of this tool. Is this going to be used on production? Is it just for the test machine? Is it going to be used by the DataTeam? Do we need permissions to run it on production? The hard codes won't work except for this limited case.